lunabot_control/include/lunabot_control/mpc.h

```diff
@@ -20,53 +22,53 @@
 #include <lunabot_control/map.h>
 
 class MPC {
-  private:
-    ros::Publisher velocity_pub_;
-    ros::Subscriber grid_sub_;
-    ros::Subscriber path_sub_;
-    ros::Subscriber robot_pos_sub_;
+private:
+  ros::Publisher velocity_pub_;
+  ros::Subscriber grid_sub_;
+  ros::Subscriber path_sub_;
+  ros::Subscriber robot_pos_sub_;
 
```

Comment:
Can convert some of the paramters below to int8 / int16 / uint or float (need to look at what specifically would be best fitting)

Reply:
wouldn't the compiler do those optimizations?

Reply:
I dont think so? Also I accidently highlighted the wrong part of the code, it should be the lines below i refer to.

Reply:
the jetson has an  Arm Cortex-A78AE cpu which will have 32bit instructions and would require additional operations to operate on smaller widths? so i think it should be fine